core/client/pom.xml

```diff
@@ -188,6 +188,12 @@
           <scope>provided</scope>
         </dependency>
         <dependency>
+          <groupId>com.google.inject</groupId>
+          <artifactId>guice</artifactId>
+          <version>3.0</version>
```

Comment:
that's not how dependency management is currently used and I think it is important to be consistent ... as much as I don't like exceptions, I am fine with making one for provided dependencies, but I would prefer if we do this consistently for all provided dependencies and not just for the presto profile